templates/provider/hmc/templates/rbac/user-facing/unmanagedcluster-editor.yaml

```diff
@@ -0,0 +1,13 @@
+# permissions for end users to edit unmanagedclusters.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    hmc.mirantis.com/aggregate-to-global-admin: "true"
```

Comment:
I think rbac considerations should be similar to managedclusters, unless it was decided otherwise